acceptance-tests/src/main/java/io/blueocean/ath/LocalDriver.java

```diff
@@ -46,106 +49,31 @@ public static void setUrlBase(String base) {
     }
 
     /**
-     * Used for callbacks in a specific browser context
+     * @return a FluentWait with default polling / timeout / exception handling
      */
-    public interface Procedure {
-        void execute() throws Exception;
+    static FluentWait<WebDriver> buildWait() {
+        return new FluentWait<>(getDriver())
+            .pollingEvery(100, TimeUnit.MILLISECONDS)
+            .withTimeout(DEFAULT_TIMEOUT, TimeUnit.MILLISECONDS)
+            .ignoring(NoSuchElementException.class);
     }
 
-    @Override
-    public void get(String s) {
-        getDriver().get(s);
-    }
-
-    @Override
-    public String getCurrentUrl() {
-        return getDriver().getCurrentUrl();
-    }
-
-    @Override
-    public String getTitle() {
-        return getDriver().getTitle();
-    }
-
-    @Override
-    public List<WebElement> findElements(By by) {
-        return new SmartWebElement(getDriver(), by).getElements();
-    }
-
-    @Override
-    public WebElement findElement(By by) {
-        return new SmartWebElement(getDriver(), by).getElement();
-    }
-
-    @Override
-    public String getPageSource() {
-        return getDriver().getPageSource();
-    }
-
-    @Override
-    public void close() {
-        WebDriver driver = getDriver();
-        if (driver != null) {
-            try {
-                driver.close();
-            } catch(Exception e) {
-                // ignore, this happens when running individual tests sometimes
-            }
+    /**
+     * Wraps the supplied element in SmartWebElement if not already a SWE
+     */
+    static WebElement wrapElement(WebDriver driver, WebElement element, By locator) {
+        if (element instanceof SmartWebElement) {
+            return element;
         }
-    }
-
-    @Override
-    public void quit() {
-        getDriver().quit();
-    }
-
-    @Override
-    public Set<String> getWindowHandles() {
-        return getDriver().getWindowHandles();
-    }
-
-    @Override
-    public String getWindowHandle() {
-        return getDriver().getWindowHandle();
-    }
-
-    @Override
-    public TargetLocator switchTo() {
-        return getDriver().switchTo();
-    }
-
-    @Override
-    public Navigation navigate() {
-        return getDriver().navigate();
-    }
-
-    @Override
-    public Options manage() {
-        return getDriver().manage();
+        return new SmartWebElement(driver, element, locator.toString());
     }
 
     /**
-     * Push a specific driver into context and execute the proc
-     * @param driver new driver in context
-     * @param proc procedure to execute
+     * Wraps the supplied elements in SmartWebElement if not already a SWE
      */
-    public static void use(WebDriver driver, Procedure proc) {
```

Comment:
I would not remove this, it can be useful to write tests that rely on multiple windows

Reply:
... still, I would not remove this